designer/client/src/components/toolbars/test/buttons/GenerateAndTestButton.tsx

```diff
@@ -14,7 +16,11 @@ function GenerateAndTestButton(props: Props) {
     const { t } = useTranslation();
     const testCapabilities = useSelector(getTestCapabilities);
     const processIsLatestVersion = useSelector(isLatestProcessVersion);
-    const available = !disabled && processIsLatestVersion && testCapabilities && testCapabilities.canGenerateTestData;
+    const available =
+        !disabled &&
+        processIsLatestVersion &&
+        testCapabilities &&
+        testCapabilities.testWithGeneratedData.status == TestCapabilityStatus.AVAILABLE;
```

Comment:
From time to time I encounter notification that there is some problem related with test capabilities. Will it help for that?